### ISSUE TITLE
Column layout don't remember the wide setting after restart

### ISSUE THREAD

**Issue by ghost** [quodlibet/quodlibet#3319]
Steps to reproduce
------------------
Remove ~/.config/quodlibet
Start Quodlibet
In "Paned Browser Preferences" check the wide box. 
Click Apply
Exit program with going to File -> Quit
Start Quodlibet
Quodlibet forgot the wide setting

Expected Output
---------------
After setting the Paned Browser Preferences to wide it should remember it after a restart

Actual Output
-------------
After setting the Paned Browser Preferences to wide it forgot the setting after a restart

Test System
-----------

### Which version of Quod Libet?
4.3.0

### Which operating system 
Arch Linux 5.5.8-arch1-1

**Comment by afontenot** [comment]
Similarly, Quod Libet forgets the column layout is forgotten if you switch to a different browsing mode and switch back.

**Comment by afontenot** [comment]
Hey, it looks like this was quietly fixed in the master without this issue getting tagged. https://github.com/quodlibet/quodlibet/commit/99aaa5a8d7a2e077f1f76366b91b85e104a270f6

Also on Arch Linux. I applied that commit on top of 4.3.0 as follows:

```
mkdir quodlibet && cd quodlibet
curl -JO "https://git.archlinux.org/svntogit/community.git/plain/trunk/PKGBUILD?h=packages/quodlibet"
makepkg -o
cd src/quodlibet-4.3.0
curl -JO "https://github.com/quodlibet/quodlibet/commit/99aaa5a8d7a2e077f1f76366b91b85e104a270f6.diff"
patch -p1 < 99aaa5a8d7a2e077f1f76366b91b85e104a270f6.diff
cd ../..
makepkg -e
```

Obviously you'll need to install any missing makedepends and add the release PGP key.

**Comment by Sr343785tyh** [comment]
> Obviously you'll need to install any missing makedepends and add the release PGP key.
doing this, get folder src with bin, lib.. then copied all of this files to /usr/*
not work for me, columns settings not saving after restart quodelibet, delete quodelibet and install quodlibet-4.2.1 again

how can i remove release PGP key which i add(gpg --recv-key 5A62D0CAB6264964)?

**Comment by declension** [comment]
Should be fixed now (4.4.0). Please re-open if not
